assignment/week2/todolist/main.js

```diff
@@ -0,0 +1,140 @@
+
+function addTodayTask(addTodayValue) {
+  //   const addTodayValue = document.querySelector("#addTodayTask").value;
+  let li_num = 0;
+  if (document.querySelector(".today-task-li") !== null) {
+    li_num += 1;
+  }
+  const li = document.createElement("li");
+  const input = document.createElement("input");
+  input.setAttribute("type", "checkbox");
+
+  const button = document.createElement("button");
+  li.setAttribute("id", addTodayValue);
+  li.setAttribute("class", "today-task-li");
+
+  const span = document.createElement("span");
+  span.innerHTML = addTodayValue;
+  span.setAttribute("class", "todo-text");
+  //   const textNode = document.createTextNode(addTodayValue);
+  if (li_num !== 0) {
+    const hr = document.createElement("hr");
+    hr.setAttribute("class", "todolist-hr");
+    li.append(hr);
+    console.log("dsahfdjksafdf");
+  }
+
+  li.appendChild(input);
+  //   li.appendChild(textNode);
+  li.appendChild(span);
+  li.appendChild(button);
+  button.innerText = "trash";
+  button.setAttribute("class", "trash-button");
+  button.addEventListener("click", deleteTasks);
+
+  document.querySelector("#todayTaskList").appendChild(li);
+}
+function addTomorrowTask(addTomorrowValue) {
+  //   const addTomorrowValue = document.querySelector("#addTomorrowTask").value;
+  let li_num = 0;
+  if (document.querySelector(".tomorrow-task-li") !== null) {
+    li_num += 1;
+  }
+
+  if (document.querySelector("li") !== null) {
+    const hr = document.createElement("hr");
+    hr.setAttribute("class", "todolist-hr");
+    console.log("dsahfdjksafdf");
+  }
+
+  const li = document.createElement("li");
+  const input = document.createElement("input");
+  input.setAttribute("type", "checkbox");
+
+  const button = document.createElement("button");
+  li.setAttribute("id", addTomorrowValue);
+  li.setAttribute("class", "tomorrow-task-li");
+
+  const span = document.createElement("span");
+  span.innerHTML = addTomorrowValue;
+  span.setAttribute("class", "todo-text");
+  if (li_num !== 0) {
+    const hr = document.createElement("hr");
+    hr.setAttribute("class", "todolist-hr");
+    li.append(hr);
+    console.log("dsahfdjksafdf");
+  }
+
+  li.appendChild(input);
+  li.appendChild(span);
+  li.appendChild(button);
+  button.innerText = "trash";
+  button.setAttribute("class", "trash-button");
+  button.addEventListener("click", deleteTasks);
+
+  document.querySelector("#tomorrowTaskList").appendChild(li);
+}
+function deleteTasks(e) {
+  const deleteTask = e.target.parentElement;
+  deleteTask.remove();
+}
+function submitTodayClick(e) {
+  e.preventDefault();
+  const addTodayValue = todayInput.value;
+  todayInput.value = "";
+  addTodayTask(addTodayValue);
+}
+function submitTomorrowClick(e) {
+  e.preventDefault();
+  const addTomorrowValue = tomorrowInput.value;
+  tomorrowInput.value = "";
+  addTomorrowTask(addTomorrowValue);
+}
+todayInput.addEventListener("keypress", (e) => {
+  if (e.keyCode === 13) {
```

Comment:
```suggestion
  if (e.key === "Enter") {
```

e.keycode는 더이상 사용이 권장되는 방식이 아니여서(=deprecated된 방식) 위와같은 방식으로 사용하면 좋을 것 같습니다!!

Reply:
헉! 몰랐던 부분을 알게 되었숩니다! 반영할게요!!